core/domain/summary_services.py

```diff
@@ -119,7 +120,7 @@ def get_learner_collection_dict_by_id(
 
     exp_ids = collection.exploration_ids
     exp_summary_dicts = get_displayable_exp_summary_dicts_matching_ids(
-        exp_ids, editor_user_id=user_id)
+        exp_ids, editor_user_id=user_id, editor_actions=user_actions)
```

Comment:
This is a case that really makes me strongly favor combining user_id + actions into a "thing", because either both should be None or both should exist, and that constraint is not clear when the args are individual ones.